src/css/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin font-weight($weight) {
+  font-weight: $weight;
+}
```

Comment:
Mixins are better used for elements that need many of the same CSS properties, not just a single one. This is actually adding more code for you rather than DRYing it up, which is a big benefit of using Sass